src/nunjucks/moduk/components/cookie-banner/__tests__/macro.test.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs'
+import path from 'path'
```

Comment:
I think we should use the `node:` protocol imports:

```suggestion
import fs from 'node:fs'
import path from 'node:path'
```

But needs updating everywhere so can sort that out separately.